tests/rustdoc-ui/check-attr-test.stderr

```diff
@@ -15,30 +15,6 @@ note: the lint level is defined here
 3 | #![deny(rustdoc::invalid_codeblock_attributes)]
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-error: unknown attribute `compilefail`. Did you mean `compile_fail`?
- --> $DIR/check-attr-test.rs:5:1
-  |
-5 | / /// foo
-6 | | ///
-7 | | /// ```compile-fail,compilefail,comPile_fail
```

Comment:
For some reason only the first of the typos on this line cause an error to be emitted now. I'm not sure why this is, but I imagine it has something to do with the difference between `struct_span_lint_hir` and `emit_spanned_lint`. I'll look into it more later...